.appveyor.yml

```diff
@@ -0,0 +1,36 @@
+# Thanks to https://github.com/starkat99/appveyor-rust
```

Comment:
you need to make an entry in the [bors](https://github.com/sodiumoxide/sodiumoxide/blob/master/bors.toml)  configuration.